index.html

```diff
@@ -20,28 +20,38 @@
         <button class="hamburger">&#9776;</button>
         <div class="nav-list">
           <ul>
-            <li><a href="#">About</a></li>
-            <li><a href="#">Projects</a></li>
-            <li><a href="#">Skills</a></li>
+            <li class="tab" data-content="aboutMe"><a href="#">About</a></li>
+            <li class="tab" data-content="projectsContainer"><a href="#">Projects</a></li>
             <li><a href="https://github.com/sheridp2" target="_blank">GitHub</a></li>
             <li><a href="https://www.facebook.com/patrick.sheridan.773?ref=bookmarks" target="_blank">Facebook</a></li>
           <ul>
         </div>
       </nav>
 
     </header>
-
-    <div class="about-container">
-      <div class="about-text">
-        <h2>About Me</h2>
-        <p>Filler Text</p>
+    <section id="aboutMe" class="tab-content">
```

Comment:
**TODO FOR LOST POINTS**

Professional HTML convention actually doesn't camel-case attributes (confusing, I know, but just got to get used to contradictory professional standards). 

Line 32 should be formatted as:

```
<section id="about-me" class="tab-content">

```